### PR TITLE
(COV-765): refactor/hideEmptyCharts: hide empty charts also add tests and storybook

### PR DESCRIPTION
**Pull request by ocshawn**
Jira Ticket: [COV-765](https://occ-data.atlassian.net/browse/COV-765)

### Breaking Changes 
- remove chartIsEmpty property from SummaryChartGroup component

### Improvements
- have empty chart message trigger when chart has no data
- add tests and storybook for new empty chart functionality

### CODE REVIEW

**Review by ZakirG — DISMISSED** [review]

**Comment by ocshawn** [comment]
per @mfshao i have made it to show no data message instead, enabling this functionality based on if no items are present. Deleted old functionality that was based on a property being set

File: src/components/charts/SummaryPieChart/index.jsx [hunk 1]
```diff
@@ -181,7 +180,6 @@ SummaryPieChart.defaultProps = {
   useCustomizedColorMap: false,
   customizedColorMap: ['#3283c8'],
   maximumDisplayItem: 15,
-  chartIsEmpty: false,
   chartEmptyMessage: 'Cannot render this chart because some fields don\'t apply',
```
Comment (mfshao) [inline src/components/charts/SummaryPieChart/index.jsx]: suggest to change this default message to something shorter, like "This chart is empty" or "No data"

you can verify how it looks in the storybook

**Review by mfshao — COMMENTED** [review]
generally look good, one small comment below and one suggestion: I think you should put in `Breaking changes` say that props `chartIsEmpty` has been removed from the `SummaryChartGroup` component

and update the package versions accordingly

**Comment by ocshawn** [comment]
changes made to default message and pull request description

**Review by mfshao — COMMENTED** [review]
can you also update the `chartEmptyMessage` for those individual chart compoenets as well? just for the sake of consistency

also where is the package version update?

**Comment by ocshawn** [comment]
Since i am working on two updates at the same time im just going to update the package after they are both done, will update the version in whichever one is last. I have made the other changes please review

**Comment by mfshao** [comment]
> Since i am working on two updates at the same time im just going to update the package after they are both done, will update the version in whichever one is last. I have made the other changes please review

I do think PR 101 still need some thoughts/discussions, but if you want to let this PR wait on that one then I also don't have any objections

**Comment by ocshawn** [comment]
I agree with  PR 101 needing a lot of work so i will release this by itself, please approve

**Review by mfshao — APPROVED** [review]
👍 🚀
